zero/radiation_read.c

```diff
@@ -1,3 +1,4 @@
+#include <gkyl_util.h>
 #include <gkyl_radiation_read.h>
```

Comment:
are there unit tests that just test radiation_read functions? do we know if it is valgrind clean?

Reply:
There aren't any unit tests for radiation_read specifically, but all of the functions in it are used by ctest_dg_rad_gyrokinetic. I think it is valgrind clean (it has 1 reachable piece of memory that says it is in acutest).